Candidate: CVE-2013-7040
PublicDate: 2013-12-09
References:
 http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2013-7040
 http://www.openwall.com/lists/oss-security/2013/12/09
Description:
 CPython hash secret can be recoved remotely, resulting in a DoS from a
 hash collision.
Ubuntu-Description:
Notes:
 mdeslaur> This is being fixed upstream by changing the hashing algorithm
 mdeslaur> to SipHash in python 3.4.
 mdeslaur> See http://www.python.org/dev/peps/pep-0456/
 mdeslaur> Upstream has no intention of backporting this change to stable
 mdeslaur> releases.
Bugs:
 http://bugs.python.org/issue14621
Priority: low
Discovered-by:
Assigned-to:

Patches_python2.7:
upstream_python2.7: needs-triage
lucid_python2.7: DNE
precise_python2.7: needed
quantal_python2.7: needed
raring_python2.7: needed
saucy_python2.7: needed
devel_python2.7: needed

Patches_python2.6:
upstream_python2.6: needs-triage
lucid_python2.6: needed
precise_python2.6: DNE
quantal_python2.6: DNE
raring_python2.6: DNE
saucy_python2.6: DNE
devel_python2.6: DNE

Patches_python3.2:
upstream_python3.2: needs-triage
lucid_python3.2: DNE
precise_python3.2: needed
quantal_python3.2: needed
raring_python3.2: DNE
saucy_python3.2: DNE
devel_python3.2: DNE

Patches_python3.3:
upstream_python3.3: needs-triage
lucid_python3.3: DNE
precise_python3.3: DNE
quantal_python3.3: needed
raring_python3.3: needed
saucy_python3.3: needed
devel_python3.3: needed

Patches_python3.1:
upstream_python3.1: needs-triage
lucid_python3.1: ignored (reached end-of-life)
precise_python3.1: DNE
quantal_python3.1: DNE
raring_python3.1: DNE
saucy_python3.1: DNE
devel_python3.1: DNE
